src/OutdatedCommand.php

```diff
@@ -71,13 +80,13 @@ public function execute(InputInterface $input, OutputInterface $output)
             }
 
             if ($outdated) {
-                $climate->br()->columns($outdated, 3)->br();
+                $climate->br()->columns($outdated, 3);
```

Comment:
We need to fix these breaks. Maybe they should always end with `->br()` instead of in the beginning. If there only are outdated packages there wont be any new line at the bottom e.g.